examples/tank_pid/tank_pid.cc

```diff
@@ -0,0 +1,277 @@
+// BoB robotics includes
+#include "robots/tank_pid.h"
+#include "common/background_exception_catcher.h"
+#include "common/main.h"
+#include "common/plot_agent.h"
+#include "common/read_objects.h"
+#include "hid/joystick.h"
+#include "vicon/udp.h"
+
+// This program can be run locally on the robot or remotely
+#ifdef NO_I2C_ROBOT
+#include "net/client.h"
+#include "robots/tank_netsink.h"
+#else
+#include "robots/norbot.h"
+#endif
+
+// Third-party includes
+#include "third_party/matplotlibcpp.h"
+#include "third_party/path.h"
+#include "third_party/units.h"
+
+// Standard C includes
+#include <cstdlib>
+
+// Standard C++ includes
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <vector>
+
+using namespace BoBRobotics;
+using namespace units::length;
+using namespace units::literals;
+using namespace units::angle;
+using namespace units::angular_velocity;
+using namespace units::velocity;
+using namespace std::literals;
+namespace plt = matplotlibcpp;
+
+// For sound
+#define PLAY_PATH "/usr/bin/play"
```

Comment:
hahah, is this really necessary? I'm thoroughly glad my machine doesn't come with that ogg file...